t/fixtures/CA/Makefile

```diff
@@ -0,0 +1,29 @@
+all: client.key client.crt server.key server.crt ca-bundle.crt:
```

Comment:
This Makefile is going to be super useful :+1:

Reply:
Yeah, got tired of figuring out how to generate them every time 😄